test/helpers/getImplementationsAndAPI.js

```diff
@@ -3,13 +3,10 @@ import dartSass from "sass";
 // eslint-disable-next-line import/no-namespace
 import * as SassEmbedded from "sass-embedded";
 
+const [nodeMajor] = process.versions.node.split(".").map(Number);
```

Comment:
The node major line is used multiple times, maybe extract it to a util file?

Reply:
We should to find working solution firstly